<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskPanel</class>
 <widget class="QWidget" name="TaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load example gm_vehicle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="MainLogo">
         <property name="minimumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>128</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../icons/Ico.xpm</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="GM_VehicleSelectionBox">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>80</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string>Select gm_vehicle example geometry</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QComboBox" name="GM_VehicleLdExamples">
            <property name="currentText">
             <string>Basic Ground Mobile Wheeled Vehicle Model</string>
            </property>
            <item>
             <property name="text">
              <string>Basic Ground Mobile Wheeled Vehicle Model</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Areil Nomad Model Raw </string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hog Jackson WP Model</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tabby  Two Seat Model</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Project 999 Placeholder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Project 998 Placeholder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Project 997 Placeholder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Project 996 Placeholder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Project 995 Placeholder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Project 994 Placeholder</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
